src/Ilicop.Web/Ilitools/IlitoolsEnvironment.cs

```diff
@@ -56,6 +63,11 @@ public class IlitoolsEnvironment
         /// </summary>
         public bool IsIli2GpkgInitialized => EnableGpkgValidation && !string.IsNullOrWhiteSpace(Ili2GpkgPath);
 
+        /// <summary>
+        /// Gets the plugins directory.
+        /// </summary>
+        public string PluginsDir => Path.Combine(HomeDir, "plugins");
```

Comment:
Q: Ist das HomeDir nicht dort wo die Tools installiert werden? Die Plugins werden doch von aussen gemountet. Müssten diese also nicht in "/plugins" liegen. Also root?


(PP: Homedir etwas bezeichnender wählen (z.B. InstallationDir)